src/components/Reports/TeamReport/TeamReport.jsx

```diff
@@ -85,6 +96,7 @@ export function TeamReport({ match }) {
       }
     } else {
       setSelectedTeams(prevSelectedTeams =>
+        // eslint-disable-next-line no-shadow
```

Comment:
Same as above. If you try your best and still cannot actually fix it, just let me know.